android/java/org/chromium/chrome/browser/crypto_wallet/activities/NetworkSelectorActivity.java

```diff
@@ -21,20 +26,25 @@
 import org.chromium.chrome.browser.crypto_wallet.adapters.NetworkSelectorAdapter;
 import org.chromium.chrome.browser.crypto_wallet.util.JavaUtils;
 import org.chromium.chrome.browser.crypto_wallet.util.Utils;
+import org.chromium.chrome.browser.settings.BraveSettingsLauncherImpl;
+import org.chromium.chrome.browser.settings.BraveWalletAddNetworksFragment;
 import org.chromium.chrome.browser.util.LiveDataUtil;
 import org.chromium.content_public.browser.UiThreadTaskTraits;
+import org.chromium.components.browser_ui.settings.SettingsLauncher;
 
 import java.util.ArrayList;
 
 public class NetworkSelectorActivity
         extends BraveWalletBaseActivity implements NetworkSelectorAdapter.NetworkClickListener {
     public static String NETWORK_SELECTOR_MODE = "network_selector_mode";
     private static final String TAG = NetworkSelectorActivity.class.getSimpleName();
+    private static final int REQUEST_CODE_ADD_NETWORK = 1;
     private NetworkSelectorModel.Mode mMode;
     private RecyclerView mRVNetworkSelector;
     private NetworkSelectorAdapter networkSelectorAdapter;
     private MaterialToolbar mToolbar;
     private String mSelectedNetwork;
+    private SettingsLauncher mSettingsLauncher = new BraveSettingsLauncherImpl();
```

Comment:
any reason why this has to be initialised in the class scope? can move it inside `initState`?

Reply:
Moving it inside `initState()`.